idris-commands.el

```diff
@@ -74,8 +74,9 @@
 
 (defun idris-ensure-process-and-repl-buffer ()
   "Ensure that an Idris process is running and the Idris REPL buffer exists."
-  (idris-repl-buffer)
-  (idris-run))
+  (idris-run)
+  (with-current-buffer (idris-repl-buffer)
+    (goto-char (point-max))))
```

Comment:
OK, I moved it to a more specific location (where it should have been all along).